### PR TITLE
Updated Storages

### PR DESCRIPTION
**Pull request by Dzierzan**
image

This pull request transforms `Storage` building into new Synaty refinery which is later renamed to `Storage` anyway.

Since we no longer use refineries, Yuruki variant is also modified (removing mainly chimneys).

So gameplay wise, players no longer have to bother with building more old silo type buildings (like it was simplified in Red Alert 2).

@Mailaender There are right now unused animations which I want to be used as unload resources animation when a tanker reaches the building. This needs custom trait.

### CODE REVIEW

File: mods/hv/rules/buildings.yaml [hunk 1]
```diff
@@ -1520,14 +1463,9 @@ REFINERY:
 		Amount: -30
 	RenderSprites:
 		PlayerPalette: green
-		Image: refinery-yi
+		Image: storage-yi
 		FactionImages:
-			sc: refinery-sc
-	WithResourceStoragePipsDecoration:
```
Comment (Mailaender) [inline mods/hv/rules/buildings.yaml:1526]: I think we need to keep this as there is no other fullness indicator in the UI.

Comment (Dzierzan) [reply mods/hv/rules/buildings.yaml:1526]: Wait, I thought we are dropping silos mechanic? Like it is in Red Alert 2, no more silos. So why pips are needed for that?

Comment (Mailaender) [reply mods/hv/rules/buildings.yaml:1526]: Well, but you kept

```salt
	StoresResources:
		Capacity: 5000
```

Comment (Dzierzan) [reply mods/hv/rules/buildings.yaml:1526]: Ohh snap!

**Pull request merged**